game/screens.rpy

```diff
@@ -471,7 +460,9 @@ init -501 screen navigation():
                     textbutton _("Main Menu") action NullAction()
 
             textbutton _("Settings") action [ShowMenu("preferences"), SensitiveIf(renpy.get_screen("preferences") == None)]
-
+            
+            #Added by AlexAzumi
+            textbutton _("Characters") action [ShowMenu("characters"), SensitiveIf(renpy.get_screen("characters") == None)]
```

Comment:
I think we should not show this button immediately after the first start: it will be a small spoiler. It is better to hide it under some condition.